configs/katello/3.9.yaml

```diff
@@ -0,0 +1,270 @@
+---
+:project: katello
+:github_org: katello
+:prior_releases:
+  :3.8.0:
+    :redmine_version_id: 356
+  :3.7.0:
+    :redmine_version_id: 264
+  :3.7.1:
+    :redmine_version_id: 355
+:releases:
+  :3.9.0:
+    :redmine_version_id: 899
+:code_name: New England IPA
+:repos:
+  :katello:
+    :main: true
+    :branch: KATELLO-3.8
+    :repo: https://github.com/Katello/katello.git
+    :version_branch: true
+  :katello-installer:
+    :branch: KATELLO-3.8
+    :repo: https://github.com/Katello/katello-installer.git
+    :version_branch: true
+  :foreman-packaging:
+    :branch: rpm/1.19
```

Comment:
Won't it be based on 1.20?

Reply:
I had not looked to see if the branch was made yet, but I had assumed not because foreman and katello were not made. I'll remove for now.